### PR TITLE
docs: homepage copy pass by Theo

### PR DESCRIPTION
**Pull request by TheoBr**
- Copy for main blurb
- More copy updating, banner and more

Closes #<issue>

## ✅ Checklist

- [x] I have followed every step in the [contributing guide](https://github.com/t3-oss/create-t3-app/blob/main/CONTRIBUTING.md) (updated 2022-10-06).
- [ ] The PR title follows the convention we established [conventional-commit](https://www.conventionalcommits.org/en/v1.0.0/)
- [x] I performed a functional test on my final commit

---

## Changelog

_[Short description of what has changed]_

---

## Screenshots

_[Screenshots]_

💯

### CODE REVIEW

File: www/src/components/landingPage/tweets/tweets.astro [hunk 1]
```diff
@@ -73,21 +73,21 @@ if (import.meta.env.MODE === "CI" || !token) {
...
     <p
       class="mt-4 text-base lg:text-lg md:text-lg text-t3-purple-200 max-w-3xl"
     >
-      Want to be listed here? Mention <a
+      Mention <a
         href="https://twitter.com/search?q=create-t3-app"
         rel="noopener noreferrer"
         class="text-t3-purple-100">#create-t3-app</a
-      > on Twitter!
+      > if you want to pop up here 😉
```
Comment (nexxeln) [inline www/src/components/landingPage/tweets/tweets.astro:90]:
```suggestion
      > if you want to pop up here ;)
```

File: www/src/components/landingPage/stack/stack.astro [hunk 2]
```diff
@@ -43,12 +43,9 @@ import Feature from "./card.astro";
...
           <Fragment slot="description">
-            We love React. It has made UI development accessible in ways we
-            never imagined before. It also can lead developers down some rough
-            paths. Next.js offers a lightly opinionated, heavily optimized
-            approach to creating applications using React. From routing to API
-            definitions to image rendering, we trust Next.js to lead developers
-            towards good decisions.
+            Next.js offers a lightly opinionated, heavily optimized approach to
+            creating applications using React. It's the industry standard and
+            we're proud to build on top of it 😊
```
Comment (nexxeln) [inline www/src/components/landingPage/stack/stack.astro:48]: No emoji please lol

Comment (TheoBr) [reply www/src/components/landingPage/stack/stack.astro:48]: What you got against emoji?

Comment (nexxeln) [reply www/src/components/landingPage/stack/stack.astro:48]: emojis on landing pages are cringe™️

**Review by nexxeln — APPROVED** [review]

**Pull request merged**